smarts/core/tests/test_sensors.py

```diff
@@ -26,3 +35,127 @@ def test_driven_path_sensor():
         assert len(sensor()) <= max_path_length
 
     sensor.teardown()
+
+
+AGENT_ID = "Agent-007"
+
+
+@pytest.fixture
```

Comment:
nit: Most of our tests have been ordered with fixtures at the top of the file, after the constants and before the tests.